pkg/local_workflows/config_utils/feature_flag.go

```diff
@@ -0,0 +1,30 @@
+package config_utils
```

Comment:
maybe move this package outside `local_workflows` if it can be used by external workflows as well?

Reply:
In general it can be used outside already as it is in `pkg`. `local_workflows` is a currently misnamed package that collects logic that is less technical platform and more re-usable domain logic. In another discussion we will try to improve this properly, until then I fall back to putting things here if this makes sense.